package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grafana-iot-twinmaker-app",
-  "version": "1.2.1",
+  "version": "1.3.0-pre",
```

Comment:
What's the reason for bumping the version now?

Reply:
🤔 -- really depends how we want to use main, but given that main is the *next* version, not the last version, i think it is better to bump sooner than later (it is weird to have dev artifacts with the same version as the previous release!)